modules/seqwish/induce/main.nf

```diff
@@ -2,11 +2,11 @@ process SEQWISH_INDUCE {
     tag "$meta.id"
     label 'process_medium'
 
-    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
-    conda (params.enable_conda ? 'bioconda::seqwish=0.7.2' : null)
+    conda (params.enable_conda ? 'bioconda::seqwish=0.7.6' : null)
+
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/seqwish:0.7.2--h2e03b76_0' :
-        'quay.io/biocontainers/seqwish:0.7.2--h2e03b76_0' }"
+        'https://depot.galaxyproject.org/singularity/seqwish:0.7.6--h5b5514e_1' :
+        'quay.io/biocontainers/seqwish: 0.7.6--h5b5514e_1' }"
```

Comment:
There is a space here

Reply:
Thanks @FriederikeHanssen! I updated accordingly, fingers crossed!